user_docs/en/userGuide.md

```diff
@@ -1653,53 +1658,107 @@ When the confirmation dialog appears, select "restart now" or "restart later" if
 
 ##### Save configuration on exit {#GeneralSettingsSaveConfig}
 
-This option is a checkbox that, when checked, tells NVDA to automatically save the current configuration when you exit NVDA.
+| . {.hideHeaderRow} |.|
+|---|---|
+|Options |enabled, disabled|
+|Default |enabled|
+
+| Option |Behaviour|
+|---|---|
+|enabled |NVDA will automatically save the current configuration when you exit.|
+|disabled |NVDA will exit directly without saving the configuration. The changes to the settings of the user configuration will be lost.|
 
 ##### Show exit options when exiting NVDA {#GeneralSettingsShowExitOptions}
 
-This option is a checkbox that allows you to choose whether or not a dialog appears when you exit NVDA that asks what action you want to perform.
-When checked, a dialog will appear when you attempt to exit NVDA asking whether you want to exit, restart, restart with add-ons disabled or install pending updates (if any).
-When unchecked, NVDA will exit immediately.
+This setting is a checkbox that allows you to choose whether or not a dialog appears when you exit NVDA that asks what action you want to perform.

```

Comment:
I think it would be better to retain longer descriptions like this.

Thoughts @Qchristensen ?